src/mozanalysis/metrics/fenix.py

```diff
@@ -0,0 +1,95 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pyspark.sql import functions as F
+
+from mozanalysis.metrics import Metric, DataSource, agg_sum
+from mozanalysis.utils import all_
+from mozanalysis.udf import extract_search_counts
+
+
+@DataSource.from_func()
+def fenix_baseline(spark, experiment):
+    bs = spark.table('org_mozilla_fenix_baseline_parquet')
+    return bs.withColumn(
+        'client_id', bs.client_info.client_id
+    ).withColumn(
```

Comment:
Unless `duration` and `first_run_date` are going to be used by lots of different metrics, I'd suggest constructing these columns as part of the relevant metric rather than the data source.